pkg/models/dashboards.go

```diff
@@ -34,7 +34,7 @@ var (
 	ErrDashboardCannotSaveProvisionedDashboard   = errors.New("Cannot save provisioned dashboard")
 	ErrDashboardRefreshIntervalTooShort          = errors.New("Dashboard refresh interval is too low")
 	ErrDashboardCannotDeleteProvisionedDashboard = errors.New("provisioned dashboard cannot be deleted")
-	ErrDashboardIdentifierNotSet                 = errors.New("Unique identfier needed to be able to get a dashboard")
+	ErrDashboardIdentifierNotSet                 = errors.New("Unique identifier needed to be able to get a dashboard")
```

Comment:
Technically this is a change, but hopefully people aren't parsing errors by string....